frameworks/Ruby/grape/Gemfile

```diff
@@ -1,20 +1,21 @@
 source 'http://rubygems.org'
 
 platforms :jruby do
-  gem "activerecord-jdbcmysql-adapter", "~> 1.3.9", :require => false
-  gem 'torqbox', '0.1.7'
-  gem "trinidad", "1.4.6"
+  gem 'activerecord-jdbcmysql-adapter', '~> 1.3.9', :require => false
+  gem 'torquebox', '3.2.0'
```

Comment:
Does this work? I'm not sure `torquebox` is a like-for-like replacement for `torqbox`.